build-aux/release.sh

```diff
@@ -34,10 +34,19 @@ mv crun-*.tar.xz $OUTDIR
 
 make distclean
 
-make -C contrib/static-builder-x86_64 build-image RUNTIME=$RUNTIME
-make -C contrib/static-builder-x86_64 build-crun CRUN_SOURCE=$(pwd) RUNTIME=$RUNTIME
+RUNTIME=${RUNTIME:-podman}
 
-mv crun $OUTDIR/crun-$VERSION-static-x86_64
+rm -rf /nix
+mkdir -p /nix
+$RUNTIME run --rm --privileged -v /:/mnt nixos/nix cp -rfT /nix /mnt/nix
+
+$RUNTIME run --rm --privileged -v /nix:/nix -v ${PWD}:${PWD} -w ${PWD} nixos/nix \
+    nix --print-build-logs --option cores 8 --option max-jobs 8 build --file nix/
+cp ./result/bin/crun $OUTDIR/crun-$VERSION-linux-amd64
+
+$RUNTIME run --rm --privileged -v /nix:/nix -v ${PWD}:${PWD} -w ${PWD} nixos/nix \
+    nix --print-build-logs --option cores 8 --option max-jobs 8 build --file nix/ --arg disableSystemd false
+cp ./result/bin/crun $OUTDIR/crun-$VERSION-linux-amd64-systemd
```

Comment:
let's call the version with systemd: $OUTDIR/crun-$VERSION-linux-amd64
and the version without: $OUTDIR/crun-$VERSION-nosystemd-linux-amd64

Reply:
or name it as `crun-linux-amd64-disableSystemd`?

Reply:
your call :-)